datamatch/CMakeLists.txt

```diff
@@ -1,12 +1,12 @@
 ################################################################################
-# Copyright (C) 2014-2019 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH  #
+# Copyright (C) 2014-2022 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH  #
 #                                                                              #
```

Comment:
In my opinion we should also deprecate the library. 

If I am correct it is not used by any of the experiments. 
This was a copy of some Panda code long time ago to make it generally available also for other experiments. It is definitely not used by CBM and and I am sure Panda still uses there own implementation (@karabowi could you please confirm).